cfme/configure/configuration/region_settings.py

```diff
@@ -974,15 +973,24 @@ def is_displayed(self):
         )
 
 
+class ReplicationKebab(Kebab):
```

Comment:
Allowing this because of the limited scope, but I'm calling it tech debt.

Whether or not the kebab options are disabled should be separate from the locator identifying them.

This allows for detecting whether something is disabled or not - with the disabled filter built into the XPATH for the item locator, in a test, there's no way of knowing if the item is there and disabled, or just completely missing. An important distinction for UI test automation.

The methods for integrating checks of 'disabled' class on the widget should be added to wt.pf.Kebab, not this individual widget.